styles/partials/base-partials/_forms.scss

```diff
@@ -0,0 +1,195 @@
+//not sure this is needed
```

Comment:
I'm weary of copying and pasting many of these from the current quartz. For one, there's not a lot of `BEM` in here. Secondly, this enforces the kind of thing that often makes components hard to implement, which is code like this:

```scss
.form {
  fieldset {
    &.inline {
      label, .inner-button, input {
        display: inline-block;
      }
    }
  }
```

I'd rather have something like:

```scss
.form--thing-foo {
  display: inline-block;
}
```

Because otherwise it's really inflexible. ie. you *must* have a `.form` with a `<fieldset class="inline">` containing a `<label>`.